src/connectedk8s/azext_connectedk8s/_params.py

```diff
@@ -46,6 +51,21 @@ def load_arguments(self, _):
         c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
         c.argument('arc_agent_version', options_list=['--agent-version'], help='Version of agent to update the helm charts to.')
 
+    with self.argument_context('connectedk8s enable-features') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+        c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to enable.')
+        c.argument('aad_client_id', options_list=['--aad-client-id'], help='AAD authorization client ID for enabling AAD RBAC. Specify when enabling azure-rbac.')
+        c.argument('aad_client_secret', options_list=['--aad-client-secret'], help='AAD authorization client secret for enabling AAD RBAC. Specify when enabling azure-rbac.')
+
+    with self.argument_context('connectedk8s disable-features') as c:
+        c.argument('cluster_name', options_list=['--name', '-n'], id_part='name', help='The name of the connected cluster.')
+        c.argument('kube_config', options_list=['--kube-config'], help='Path to the kube config file.')
+        c.argument('kube_context', options_list=['--kube-context'], help='Kubconfig context from current machine.')
+        c.argument('features', features_types, options_list=['--features'], help='Space-separated list of features you want to disable.')
+        c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.')
```

Comment:
Instead of us creating this param, lets use az cli core for this prompt. Check commands.py for connectedk8s delete

Reply:
confirmation=True

Reply:
That doesn't allow custom messaging based on user inputs, this scheme is also used in _'az aks'_
https://github.com/Azure/azure-cli-extensions/blob/955bf2093d0f181592a640ba1b8099be4a5f5afb/src/aks-preview/azext_aks_preview/_params.py#L159

Reply:
We use it like this : 
```
confirmation_message = "Disabling few of the features may adversely impact dependent resources. Learn more about this at https://aka.ms/ArcK8sDependentResources. \n" + "Are you sure you want to disable these features: {}".format(features)
utils.user_confirmation(confirmation_message, yes)
```

Reply:
Got it